testing/test/install_relocatable-complex/CMakeLists.txt

```diff
@@ -0,0 +1,25 @@
+#=============================================================================
+# Copyright (c) 2022-2023, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#=============================================================================
+cmake_minimum_required(VERSION 3.20)
+set(CMAKE_CONFIGURATION_TYPES Debug)
+project(rapids-test-project LANGUAGES CUDA)
+
+include(${rapids-cmake-dir}/rapids-test.cmake)
+
+add_library(preload SHARED usage.cu)
```

Comment:
The `rapids_test_init` will enable the CUDA language so that it can do the gpu resource spec generation. So by using CUDA language here we avoid the runtime cost of enabling an extra languages